force-app/main/default/flows/Before_Save_Session_Registration_Validations.flow-meta.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>54.0</apiVersion>
+    <assignments>
```

Comment:
Because Session and Attendee can not be changed on Session Registration, we don't need to check "event registration" on update operation @bmtemakina